.travis.yml

```diff
@@ -54,12 +54,14 @@ script:
   - mkdir -p "$HOME/.ipython/profile_default"
   - "echo 'c.HistoryAccessor.enabled = False\n' > $HOME/.ipython/profile_default/ipython_config.py"
   - export TEST_EXTRA=""
-  - if [[ "$COVERAGE" == "true" ]]; then
-      export TEST_EXTRA="--cov-config .coveragerc --cov nengo";
-    elif [[ "$STATIC" == "true" ]]; then
+  - if [[ "$STATIC" == "true" ]]; then
       flake8 -v nengo;
       pylint nengo;
     else
+      if [[ "$COVERAGE" == "true" ]]; then
+        export TEST_EXTRA="--cov-config .coveragerc --cov nengo";
+      fi
+
```

Comment:
Huh, I'm surprised TravisCI will let you have blank lines like this. good to know!